packages/ws-worker/test/reasons.test.ts

```diff
@@ -236,7 +239,27 @@ test('exception: failed to load credential', async (t) => {
 
   t.is(reason.reason, 'exception');
   t.is(reason.error_type, 'CredentialLoadError');
-  t.is(reason.error_message, 'Failed to load credential zzz: err');
+  t.is(
+    reason.error_message,
+    'Failed to load credential zzz: [fetch:credential] Error: err'
```

Comment:
I don't understand this diff... nothing has changed here, so why is this error message different (and worse?)

Reply:
It think it's just changed how the event bubbles out of the mock. I'm OK with it. There's no diff in prod here.